test/xpu_api/utilities/pairs/pairs.pair/piecewise.pass.cpp

```diff
@@ -0,0 +1,61 @@
+// -*- C++ -*-
```

Comment:
It seems like this test duplicates `piecewise_construct.pass.cpp`

Reply:
Done: test deleted.